benchmark/benchmark_hi_res_hydrostatic.jl

```diff
@@ -2,8 +2,8 @@ using Oceananigans
 using SeawaterPolynomials: TEOS10EquationOfState
 using CUDA
 
-function ten_steps!(model, Δt=1e-3)
-    for _ in 1:10
+function many_steps!(model, Nt; Δt=1e-3)
+    for _ in 1:Nt
         time_step!(model, Δt)
```

Comment:
I would use NVTX here to make sure the complete time step shows in the nvidia profiler report
```suggestion
function many_steps!(model, Nt; Δt=1e-3)
    for _ in 1:Nt
        NVTX.@range "time step" begin
            time_step!(model, Δt)
        end
```